framework/wazuh/core/tests/test_agent.py

```diff
@@ -561,21 +564,21 @@ def test_agent_get_key_ko(socket_mock, send_mock):
 @patch('wazuh.core.wdb.WazuhDBConnection._send', side_effect=send_msg_to_wdb)
 @patch('socket.socket.connect')
 def test_agent_restart(socket_mock, send_mock, mock_queue):
-    """Tests if method restart calls other methods with correct params"""
+    """Test if method restart calls other methods with correct params."""
     with patch('wazuh.core.agent.Agent.getconfig', return_value={'active-response': {'disabled': 'no'}}) as \
             mock_config:
         agent = Agent(0)
         agent.restart()
 
         # Assert methods are called with correct params
-        mock_config.assert_called_once_with('com', 'active-response')
+        mock_config.assert_called_once_with('com', 'active-response', 'Wazuh v3.9.0')
```

Comment:
I see Wazuh vX.Y.Z in some places and just vX.Y.Z, please unify, i would just use vX.Y.Z

Reply:
Unification: I have used Wazuh vX.Y.Z in all places.